src/content/docs/apm/agents/net-agent/net-agent-api/net-agent-api.mdx

```diff
@@ -634,31 +614,20 @@ The following list contains the different calls you can make with the API, inclu
               </td>
 
               <td>
-                Required. Source of incoming Trace Context headers.
-              </td>
-            </tr>
-
-            <tr>
-              <td>
-                `getter`
-
-                _Func&lt;T, string, IEnumerable&lt;string>>_
-              </td>
-
-              <td>
-                Required. Caller-defined function to extract header data from the carrier.
+                Required. A key/value pair store where distributed tracing data is inserted. This needs to exist in whatever message object is being passed from the calling service to the called service, via whatever transport is being used.  For example, for Azure Service Bus messages, the `ServiceBusMessage` type has an [`ApplicationProperties` property](https://learn.microsoft.com/en-us/dotnet/api/azure.messaging.servicebus.servicebusmessage.applicationproperties?view=azure-dotnet#azure-messaging-servicebus-servicebusmessage-applicationproperties) which is an `IDictionary<string,object>`.
+                When you want to implement custom distributed tracing for an uninstrumented communications channel (e.g. a message queue), you need to find out what key/value pair storage (carrier) option exists for that channel. 
```

Comment:
We don't use e.g., either "for example" or "such as" https://docs.newrelic.com/docs/style-guide/word-choice/usage-dictionary/#eg